File: SpeedTesting/SpeedTesting.go
package main

import (
	"fmt"
	"time"
)

func parseTime(time int64) string {
	return fmt.Sprintf("%.2f", (float64(time)/1000000)) + "ms"
}

func main() {
	var n int = 10000000
	var initTime, testTime int64 = 0, 0

	// Initialize the test
	initTime = time.Now().UnixNano()
	var arr []int = make([]int, n)
	for i := 0; i < n; i++ {
		arr[i] = i
	}
	initTime = time.Now().UnixNano() - initTime

	// Run the test
	testTime = time.Now().UnixNano()
	for i := 0; i < n; i++ {
		arr[i] *= 2
	}
	testTime = time.Now().UnixNano() - testTime

	// Print the results
	fmt.Println("Init time: ", parseTime(initTime))
	fmt.Println("Test time: ", parseTime(testTime))
}

File: SpeedTesting/SpeedTesting.js
let n = 1000000,
    initTime = 0,
    testTime = 0;

// Initialize the test
initTime = performance.now();
const arr = [];
for (let i = 0; i < n; i ++) {
    arr.push(n);
}
initTime = performance.now() - initTime;

// Run the test
testTime = performance.now();
for (let i = 0; i < n; i ++) {
    arr[i] *= 2;
}
testTime = performance.now() - testTime;

// Print the results
console.log(`Init time: ${initTime.toFixed(2)}ms`);
console.log(`Test time: ${testTime.toFixed(2)}ms`);